e2e/tests/codegen/test_repo/codegen/BUILD_FILE.plz

```diff
@@ -15,15 +15,17 @@ filegroup(
     name = "provider",
     srcs = ["foo.proto"],
     provides = {
-        "go": ":srcs",
+        "go": [":srcs"],
     },
 )
 
 filegroup(
```

Comment:
Can we add a test where a target provides two values?

Reply:
Done